docs/includes/eloquent-models/PlanetSchemaVersion.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use MongoDB\Laravel\Eloquent\HasSchemaVersion;
+use MongoDB\Laravel\Eloquent\Model;
+
+class Planet extends Model
+{
+    use HasSchemaVersion;
+
+    public const SCHEMA_VERSION = 2;
+
+    public function migrateSchema(int $fromVersion): void
```

Comment:
You can add a comment that people will copy in their code: "migrate the document to the last schema version when a document with an older schema version is retrieved from the database"